src/Data/Nullable.js

```diff
@@ -1,13 +1,12 @@
 /* eslint-disable no-eq-null, eqeqeq */
 
-"use strict";
+const nullImpl = null;
+export { nullImpl as null };
```

Comment:
Can this be:

```suggestion
export const nullImpl = null;
```

or is the workaround necessary?

Reply:
Well, `null` is exported as is in PureScript:
```purs
module Nullable (null) where

foreign import null :: forall a. Nullable a
```
So, if I changed it to `export const nullImpl = null;`, then the PureScript code needs to be changed, too:
```purs
module Nullable (null) where

null :: forall a. Nullable a
null = nullImpl

foreign import nullImpl :: forall a. Nullable a
```
So, the change I've made is the lesser diff.